src/components/ui/time-formart/time-format-consts.ts

```diff
@@ -0,0 +1,9 @@
+export const LocaleTimeFormats = {
```

Comment:
I think that all the locale data should be in one file that defines a locale "bundle".

Reply:
I agree. Will push an update.

Reply:
@imdfl Do you think this data should in the `LocaleInfo` object? e.g.

```
export const LocaleInfo: Record<LocaleId, ILocaleInfo> = {
	en: {
		direction: "ltr",
		timestamp: {
			format: "MM-dd-yyyy",
			delim: "-"
		}
	},
.
.
.
```